js/app.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+(function() {
+  angular
+  .module('radio',[
+    'ui.router',
+    'ngResource',
+    'ng-token-auth',
+    'songs'
+  ])
+  .config([
+    '$stateProvider',
+    RouterFunction
+  ])
+  .config([
```

Comment:
If you have multiple `.config` and feel like being super-Angular-y, you can put them in separate files as you would with controllers/directives/etc.